pkg/resource/group/sdk.go

```diff
@@ -206,10 +204,6 @@ func (rm *resourceManager) sdkCreate(
 	}
 
 	rm.setStatusDefaults(ko)
-	if err := rm.syncPolicies(ctx, &resource{ko}); err != nil {
-		return nil, err
-	}
```

Comment:
The problem with this is that policies will never be set on the Group unless the group is updated... or until the default 10 hours elapses and the resource is requeued.

Reply:
@jaypipes just made some tests and that's true.. (will have to review few other controllers that might be missing this).